nixos/modules/tasks/lvm.nix

```diff
@@ -89,7 +89,7 @@ in {
     (mkIf cfg.boot.vdo.enable {
       boot = {
         initrd = {
-          kernelModules = [ "kvdo" ];
+          kernelModules = [ "dm-vdo" ];
```

Comment:
Shouldn't we have an assertion that if this is enabled the kernel is a recent enough version?